helpers.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Datadog API for Go
+ *
+ * Please see the included LICENSE file for licensing information.
+ *
+ * Copyright 2016 by authors and contributors.
+ */
+
+package datadog
+
+import "encoding/json"
+
+// Bool is a helper routine that allocates a new bool value
+// to store v and returns a pointer to it.
+func Bool(v bool) *bool { return &v }
+
+// GetBool is a helper routine that returns a boolean representing
+// if a value was set, and if so, dereferences the pointer to it.
+func GetBool(v *bool) (b bool, ok bool) {
+	if v != nil {
+		return *v, true
+	}
+
+	return b, ok
```

Comment:
I know that semantically this returns the zero-value for these which is equivalent to `return false, false` but it seems a bit non-standard to do it this way. What do you think about explicit `return false, false` here?

Or, I could possibly see `return b, false` but at least the `ok` should be defined?